docs/MigrationGuide.md

```diff
@@ -7,6 +7,8 @@ To see the biggest differences please consult the [changelog](Changelog.md).
 
 ### Code API changes
 
+* [#TODO](https://github.com/TouK/nussknacker/pull/TODO) Changed AdditionalUIConfigProvider to be more in line with FragmentParameter (TODO more explanation?)
```

Comment:
i thought that 1.13 was already cut off/frozen, but yes it would be simpler if this PR would be part of 1.13

what's the procedure for merging into release branches, do I just duplicate this PR to 1.13?

Reply:
It is always an open question, if we should add sth to the cut-offed branch. The Release Manager should consider the situation. The big advantage of merging it to 1.13 is that we won't change the API to often. We actually change the API too often as you can see in the migration guide, but we should try to this rarely. The disadvantage is that Release Manager might do some tests again. Just talk with RM and ask him, what does he think about that concept.